crates/meilisearch/tests/tasks/mod.rs

```diff
@@ -1038,16 +1036,15 @@ async fn test_summarized_task_deletion() {
 
 #[actix_web::test]
 async fn test_summarized_dump_creation() {
-    let server = Server::new().await;
+    let server = Server::new_shared();
     let (task, _status_code) = server.create_dump().await;
     server.wait_task(task.uid()).await;
     let (task, _) = server.get_task(task.uid()).await;
-    assert_json_snapshot!(task,
-        { ".details.dumpUid" => "[dumpUid]", ".duration" => "[duration]", ".enqueuedAt" => "[date]", ".startedAt" => "[date]", ".finishedAt" => "[date]" },
+    snapshot!(task,
```

Comment:
My bad, we cannot do it here for the dump 🙈 

Or you could update the Display implementation of our custom Value type to ignore the field `. details.dumpUid`